exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/AggregateByLabelMetricTimeSeriesBuilder.java

```diff
@@ -86,7 +86,7 @@ public void recordPoint(MetricData metric, DoublePoint point) {
     return TimeSeries.newBuilder()
         .setMetric(mapMetric(labels, descriptor.getType()))
         .setMetricKind(descriptor.getMetricKind())
-        .setResource(mapResource(projectId));
+        .setResource(mapResource(metric.getResource().getAttributes(), projectId));
```

Comment:
(MInor) Nit:  Since this is called "mapResource" shouldn't it take the `Resource` and return `MonitoredResource`?